lib/taskana-test-api/src/main/java/pro/taskana/testapi/OracleSchemaHelper.java

```diff
@@ -0,0 +1,56 @@
+package pro.taskana.testapi;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import javax.sql.DataSource;
+
+import pro.taskana.common.api.exceptions.SystemException;
+
+public final class OracleSchemaHelper {
+
+  private static final String DEFAULT_PASSWORD = "testPassword";
+
+  private OracleSchemaHelper() {
+    // hide implicitpublic one
+  }
+
+  public static void initOracleSchema(DataSource dataSource, String schemaName)
+      throws SystemException {
+    try (Connection connection = dataSource.getConnection()) {
+      Connection conn =
+          DriverManager.getConnection(
+              connection.getMetaData().getURL(), "SYSTEM", DEFAULT_PASSWORD);
+      try (Statement stmt = conn.createStatement()) {
+        stmt.execute("GRANT ALL PRIVILEGES TO TEST_USER");
+
+        stmt.execute(
+            "create tablespace "
+                + schemaName
+                + " datafile '"
+                + schemaName
+                + ".dat' size 5M autoextend on NEXT 5M MAXSIZE UNLIMITED");
+        stmt.execute(
+            "create temporary tablespace "
+                + schemaName
+                + "_TMP tempfile '"
+                + schemaName
+                + "_tmp.dat' size 5M autoextend on NEXT 5M MAXSIZE UNLIMITED");
+        stmt.execute(
+            "create user "
+                + schemaName
+                + " identified by "
+                + DEFAULT_PASSWORD
+                + " default tablespace "
+                + schemaName
+                + " temporary tablespace "
+                + schemaName
+                + "_TMP");
+        stmt.execute("ALTER USER " + schemaName + " quota unlimited on " + schemaName);
+        stmt.execute("GRANT UNLIMITED TABLESPACE TO " + schemaName);
```

Comment:
String format for readability? Or is this not useful?

Reply:
do you really think it's more readable when using string.format?

Reply:
I migrated to string.format